src/printer.rs

```diff
@@ -226,6 +226,14 @@ impl<'a> Printer<'a> {
                 // Generate error workflow command in GitHub Actions format
                 // https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message
                 diagnostics.messages.iter().for_each(|message| {
+                    println!(
+                        "file={},line={},col={},endLine={},endColumn={}",
```

Comment:
GitHub Actions Only uses these options for annotation. You probably need to add the file path message `::({})`

Reply:
What I'm trying to resolve is that without this change, in the CI logs, you _only_ see the message:

<img width="544" alt="Screen Shot 2023-01-06 at 12 25 36 PM" src="https://user-images.githubusercontent.com/1309177/211064954-f35aaec7-4087-4e03-9919-eb123ff514a5.png">

What I'd like is to surface the annotations, but also show the correct info in the logs. Any ideas?

Reply:
Ref: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-notice-message

Reply:
Probably replacing `message.kind.body(),` with  something like this will work: `format!("{}: {}", relativize_path(Path::new(&message.filename)), message.kind.body())`

Reply:
Or separately printing the file names without using the options will work